README.md

```diff
@@ -57,11 +54,10 @@ The following information needs to be provided about your widget:
 * license
 * version
 * author
-* Github username (optional)
 
 You can press \<Enter\> if you want to use the default values.
```

Comment:
Consider the re-write: Press \<Enter\> if you want to skip and use default values.

Reply:
Done